docs/02-for-app-authors/02-requirements.md

```diff
@@ -30,6 +33,87 @@ Every repository must contain a manifest describing how to build the application
 
 This manifest may import other manifest files in the same repository. It's quite common to add the [shared-modules](shared-modules) repository as a submodule.
 
+#### Example manifest
+
+This assumes that upstream brings appdata, icons and desktop files. So we will `install` them. If not, you can add them to your repo and and add them in the `sources` section.
```

Comment:
Heavily recommend pushing upstream to add the metadata too. It shouldn't be normal to have Flathub dealing with them.

Reply:
Even more a .desktop that is not upstream it's a bug. So are the icons.